server/src/main/java/com/cloud/resource/RollingMaintenanceManagerImpl.java

```diff
@@ -114,12 +115,15 @@ public boolean configure(String name, Map<String, Object> params) throws Configu
         return true;
     }
 
-    private void updateCluster(long clusterId, String state) {
+    private void updateCluster(long clusterId, String allocationState) {
```

Comment:
@nvazquez can you please take a look at it?

I changed API parameter handling from separate parameters on the method call to an object of UpdateClusterCmd class. Therefore, I needed to change this tiny piece of RollingMaintenanceManagerImpl.

Reply:
Thanks @GabrielBrascher, looks good to me. The change needed in RollingMaintenanceManagerImpl will be just the same as you've done below